VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -71,11 +72,12 @@ private void init() {
         injector = RoboGuice.getOrCreateBaseApplicationInjector((Application) this, RoboGuice.DEFAULT_STAGE,
                 (Module) RoboGuice.newDefaultRoboModule(this), (Module) new EdxDefaultModule(this));
 
+        final Config config = injector.getInstance(Config.class);
 
-        Config config = injector.getInstance(Config.class);
         // initialize Fabric
         if (config.getFabricConfig().isEnabled() && !BuildConfig.DEBUG) {
             Fabric.with(this, new Crashlytics());
+            EventBus.getDefault().register(new CrashlyticsCrashReportObserver());
```

Comment:
> you could also achieve the same effect via a static utility method and a static flag for enabling it.

I was *very* close to doing exactly that, but I liked having the decoupling of the Logger from Crashlytics API. If I didn't use the event bus, I'd probably use a static callback you can set to keep them decoupled.